jdaviz/app.py

```diff
@@ -1020,13 +1030,8 @@ def get_subsets(self, subset_name=None, spectral_only=False,
                 else:
                     all_subsets[label] = subset_region
 
-        # can this be done at the top to avoid traversing all subsets if only
-        # one is requested?
-        all_subset_names = [subset.label for subset in dc.subset_groups]
-        if subset_name and subset_name in all_subset_names:
+        if subset_name:
```

Comment:
I don't think this `if` is needed at all now. You can just `return all_subsets` no matter what, right?

Reply:
Well, then it would still be a single-item list in the single subset instead of just the subset itself.

Reply:
Ah, ok. Nvm.